mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/ProfileAssociationDao.java

```diff
@@ -53,15 +55,17 @@
    * Returns 'detail' profiles linked to 'master' profile id
    *
    * @param masterId 'master' profile id
+   * @param tenantId tenant id
    * @return future
    */
-  Future<D> getDetailProfilesByMasterId(String masterId);
+  Future<List<ChildSnapshotWrapper>> getDetailProfilesByMasterId(String tenantId, String masterId);
```

Comment:
Please move 'tenantId' argument at the end of method arguments list

Reply:
Done